src/mlpack/core/data/map_policies/missing_policy.hpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file missing_policy.hpp
+ * @author Keon Kim
+ *
+ * Missing map policy for dataset info.
+ */
+#ifndef MLPACK_CORE_DATA_MAP_POLICIES_MISSING_POLICY_HPP
+#define MLPACK_CORE_DATA_MAP_POLICIES_MISSING_POLICY_HPP
+
+#include <mlpack/core.hpp>
+#include <unordered_map>
+#include <boost/bimap.hpp>
+#include <mlpack/core/data/map_policies/datatype.hpp>
+#include <limits>
+
+
+using namespace std;
+
+namespace mlpack {
+namespace data {
+
+/**
+ * Same as increment map policy, but does not change type of features.
```

Comment:
Could you detail what the class does a little more?  I think that, based on reading it, `MissingPolicy` takes a set of tokens and maps those tokens to `NaN`---but it also maps any other tokens that can't be converted to numbers to `NaN`.  Is that correct?  If so, we should document it better, I think.